libcontainer/cgroups/utils_test.go

```diff
@@ -190,3 +192,56 @@ func BenchmarkGetCgroupMounts(b *testing.B) {
 		}
 	}
 }
+
+func TestParseCgroupString(t *testing.T) {
+	testCases := []struct {
+		input          string
+		expectedError  error
+		expectedOutput map[string]string
+	}{
+		{
+			// Taken from a CoreOS instance running systemd 225 with CPU/Mem
+			// accounting enabled in systemd
+			input: `9:blkio:/
+8:freezer:/
+7:perf_event:/
+6:devices:/system.slice/system-sshd.slice
+5:cpuset:/
+4:cpu,cpuacct:/system.slice/system-sshd.slice/sshd@126-10.240.0.15:22-xxx.yyy.zzz.aaa:33678.service
+3:net_cls,net_prio:/
+2:memory:/system.slice/system-sshd.slice/sshd@126-10.240.0.15:22-209.251.245.232:33678.service
```

Comment:
Indeed they were equal and it was doctored to remove my public ip (not that it really matters). Edited the remaining instance of it for consistency.

I prefer using close-to-real data for unit tests here since it's easy to do so.